common/src/main/java/org/spine3/samples/lobby/common/util/aggregate/AbstractLobbyAggregate.java

```diff
@@ -30,26 +30,40 @@
 import java.util.List;
 
 /**
+ * Abstract type of aggregate used in the project.
+ * Provides some tools for testing needs.
+ *
+ * @see Aggregate
  * @author Dmytro Dashenkov
  */
 @SuppressWarnings("AbstractClassWithoutAbstractMethods")
-public abstract class LobbyAggregate<I, S extends Message, B extends Message.Builder> extends Aggregate<I, S, B> {
+public abstract class AbstractLobbyAggregate<I, S extends Message, B extends Message.Builder> extends Aggregate<I, S, B> {
 
     /**
      * Creates a new aggregate instance.
      *
      * @param id the ID for the new aggregate
      * @throws IllegalArgumentException if the ID is not of one of the supported types
```

Comment:
Please explain what the supported types are. Or provide a link to the related functionality (`@see` or `@link`).